Cargo.lock

```diff
@@ -3549,7 +3554,7 @@ dependencies = [
  "socket2",
  "widestring",
  "windows-sys 0.48.0",
- "winreg",
+ "winreg 0.50.0",
```

Comment:
Check if we can use only a single version

Reply:
https://github.com/denoland/deno/pull/24108